lib/decidim/homepage_proposals/version.rb

```diff
@@ -7,7 +7,7 @@ def self.version
     end
 
     def self.decidim_version
-      "0.26"
+      "0.26.10"
```

Comment:
```suggestion
      "0.26"
```



Is the 0.26.10 version a requirement ? We currently load it in `*.gemspec` as follow : `s.add_dependency "decidim-core", "~> #{Decidim::HomepageProposals.decidim_version}"`